### ISSUE TITLE
Generating histogram from Neuroglancer fails

### ISSUE THREAD

**Issue by cnzqy1** [scalableminds/webknossos-connect#91]
When viewing any Neuroglancer precomputed dataset an error message pops up:

> Error: Requested URL /data/datasets/GintyLab/fib_test1/layers/fib_test1/histogram not found

For WKW datasets it looks like this histogram is automatically generated, but this seems to fail for Neuroglancer precomputed datasets. Similar behavior confirmed in other publicly available Neuroglancer datasets such as `harris2015`.

The error log from the webknossos-connect Docker container is the following:

```
[ERROR] Exception occurred while handling uri: 'http://127.0.0.1:8000/data/datasets/GintyLab/fib_test1/layers/fib_test1/histogram?token=tfpPS8DXcP8c3wzXgEkZ2w'
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/sanic/app.py", line 736, in handle_request
    handler, args, kwargs, uri = self.router.get(request)
  File "/usr/local/lib/python3.7/site-packages/sanic/router.py", line 396, in get
    return self._get(request.path, request.method, "")
  File "/usr/local/lib/python3.7/site-packages/sanic/router.py", line 458, in _get
    raise NotFound("Requested URL {} not found".format(url))
sanic.exceptions.NotFound: Requested URL /data/datasets/GintyLab/fib_test1/layers/fib_test1/histogram not found
```

**Comment by hotzenklotz** [comment]
Thank for opening this issue. Histograms are only supported for `*.wkw` files at the moment.

**Comment by cnzqy1** [comment]
Thanks for your reply! If implementing histogram generation for non `.wkw` files would be a lot of effort, maybe hiding the histogram panel and not attempting to generate histogram (to suppress the error) could be a stopgap?

**Comment by fm3** [comment]
Closed by PR #96
